stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2619,7 +2670,7 @@ internal class _TypedNative${Self}Buffer<${TypeParameters}> :
   }
 
   override internal init(_doNotCallMe: ()) {
-    _sanityCheckFailure("Only initialize these by calling create")
+    _sanityCheckFailure("Only create this by calling Storage.createUnhashable")
```

Comment:
Why is this `init` needed? A comment should say. Also, this should probably call `Builtin.unreachable` if that's not already part of `_sanityCheckFailure`